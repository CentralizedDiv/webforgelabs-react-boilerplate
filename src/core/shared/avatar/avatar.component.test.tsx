import React from 'react';
import STAvatar from './avatar.component';
import { shallow } from 'enzyme';

describe('STAvatar', () => {
  const component = shallow(<STAvatar/>);

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });

});