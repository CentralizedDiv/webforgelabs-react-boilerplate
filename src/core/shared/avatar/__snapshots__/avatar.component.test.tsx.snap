// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`STAvatar renders properly 1`] = `
<div
  className="st-avatar"
  onClick={[Function]}
>
  <Avatar
    shape="circle"
    size={40}
  >
    <span />
  </Avatar>
</div>
`;
